inc/saiswitch.h

```diff
@@ -2873,6 +2879,32 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_HOSTIF_OPER_STATUS_UPDATE_MODE,
 
+    /**
+     * @brief VOQ chassis isolation setting.
+     *
+     * This attribute is for VOQ Chassis only
+     * This attribute is for configuring threshold value of active fabric ports below which VOQ Chassis isolation mechanism will trigger. Value of 0 means this attribute is not considered by switch.
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     * @validonly SAI_SWITCH_ATTR_TYPE == SAI_SWITCH_TYPE_VOQ
+     */
+    SAI_SWITCH_ATTR_FABRIC_PORT_THRESHOLD_LOW,
+
+    /**
+     * @brief VOQ chassis isolation setting.
+     *
+     * This attribute is for VOQ Chassis only
+     * This attribute is for configuring threshold value of active fabric ports above which VOQ Chassis isolation mechanism will be aborted or VOQ chassis will be eligible to become part of dis-aggregated chassis. Value of 0 means this attribute is not considered by switch.
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     * @validonly SAI_SWITCH_ATTR_TYPE == SAI_SWITCH_TYPE_VOQ
+     */
+    SAI_SWITCH_ATTR_FABRIC_PORT_THRESHOLD_HIGH,
```

Comment:
Would it be easier to follow if below are renamed as:

SAI_SWITCH_ATTR_FABRIC_PORT_THRESHOLD_LOW => SAI_SWITCH_ATTR_FABRIC_PORT_ISOLATE_THRESH
SAI_SWITCH_ATTR_FABRIC_PORT_THRESHOLD_HIGH => SAI_SWITCH_ATTR_FABRIC_PORT_UNISOLATE_THRESH

If we don't want to tie the action (isolate/unisolate) to the threshold configuration, then we could consider naming it as SAI_SWITCH_ATTR_FABRIC_PORT_UP_THRESHOLD_LOW, SAI_SWITCH_ATTR_FABRIC_PORT_UP_THRESHOLD_HIGH i.e. add 'UP' since this is counting how many FABRIC ports are UP?

Reply:
Please look at the latest diffs and let me know what you think